src/common/options.js

```diff
@@ -132,7 +134,7 @@ function buildOptionsPage() {
           .append($('<input>', { type: 'checkbox', id: setting.name, name: setting.name, 'aria-describedby': setting.name + 'HelpBlock' }))
           .append($('<div>', { class: 'option-description' })
             .append($('<label>', { for: setting.name, text: setting.title }))
-            .append($('<span>', { id: setting.name + 'HelpBlock', class: 'help-block', text: setting.description }))))
+            .append($('<span>', { id: setting.name + 'HelpBlock', class: 'help-block', html: description }))))
```

Comment:
Makes sense. Markdown is a great idea. Showdown is pretty lightweight and wouldn't be a lot of overhead. Let me look into that.